src/base/bittorrent/session.cpp

```diff
@@ -2982,6 +3020,14 @@ bool Session::hasPerTorrentRatioLimit() const
     return false;
 }
 
+bool Session::hasPerTorrentSeedingTimeLimit() const
+{
+    foreach (TorrentHandle *const torrent, m_torrents)
+        if (torrent->seedingTimeLimit() >= 0) return true;
+
```

Comment:
>(only express our opinion).

This is exactly what I mean.

>Remember, everything I have done is because somewhere else in the same file is like that and it would look inconsistent. 

To be clear, I'm not forcing a change here.
Personal thought: I think the inconsistent of style isn't that critical anyway, unless someone rewrite the old code, it probably will stay there forever, so just focus on making the new code nice is enough...